### DISCUSSION TITLE
Midi expander

### DISCUSSION THREAD

**Discussion by elroy-croyden** [eh2k/squares-and-circles#81] [Q&A]
Thanks for the squares and circles, awesome work. More a question than issue, I'm currently building the midi-trig adapter as a little standalone unit. I'm assuming the 5V would typically come from the Eurorack psu. Seeing as it's standalone I'm going to use a 5V USB wallwart, to your knowledge will that work?

**Comment by eh2k** [comment]
Hi @elroy-croyden,

I draw the 5V directly from the Eurorack power supply. If your PSU does not provide 5V, you would have to add an LM7805 to the circuit.
